spec/db/data/create_join_table_rows_for_single_site_roles_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+require_relative "../../../db/data/20230706184950_create_join_table_rows_for_single_site_roles"
+
+describe "CreateJoinTableRowsForSingleSiteRoles" do
+  let(:site) { create(:site) }
+  let(:second_site) { create(:site) }
+
+  let!(:team_member) { create :user, role: create(:team_member_role, sites: [site]) }
+  let!(:legacy_team_member) { create :user, role: create(:team_member_role, legacy_vita_partner: site, sites: []) }
+  let!(:another_team_member) { create :user, role: create(:team_member_role, sites: [second_site]) }
+  let!(:site_coordinator) { create :user, role: create(:site_coordinator_role, sites: [site]) }
+  let!(:legacy_site_coordinator) { create :user, role: create(:site_coordinator_role, legacy_vita_partner: site, sites: []) }
+
+  it "resets the set of answers for each filer to nil if the whole set (including prefer not to answer) was true" do
```

Comment:
i think the text in this "it" is probably a copy/paste?

Reply:
oopsie doopsie